chart/compass/values.yaml

```diff
@@ -212,26 +212,71 @@ global:
               max_delay: 2000ms
 
   tenantFetcher:
-    enabled: false
-    manageSecrets: true
-    providerName: "compass"
-    schedule: "*/5 * * * *"
-    oauth:
-      client: ""
-      secret: ""
-      tokenURL: ""
-    endpoints:
-      tenantCreated: "127.0.0.1/events?type=created"
-      tenantDeleted: "127.0.0.1/events?type=deleted"
-      tenantUpdated: "127.0.0.1/events?type=updated"
-    fieldMapping:
-      idField: "id"
-      nameField: "name"
-      discriminatorField: ""
-      discriminatorValue: ""
-    dbPool:
-      maxOpenConnections: 1
-      maxIdleConnections: 1
+    # job1:
```

Comment:
The reason is that if there is something in here - it will be taken as a default. The problem is that we don't know what will be the default name of the production jobs, so there are only commented examples of two jobs.